planner/core/partition_pruning_test.go

```diff
@@ -98,3 +87,181 @@ func (s *testPartitionPruningSuite) TestCanBePrune(c *C) {
 	// Because unix_timestamp('2008-05-01 00:00:00') is fold to constant int 1564761600, and compare it with timestamp (report_updated)
 	// need to convert 1564761600 to a timestamp, during that step, an error happen and the result is set to <nil>
 }
+
+func (s *testPartitionPruningSuite) TestPruneUseBinarySearch(c *C) {
```

Comment:
it's better to add a test case with `maxvalue = true`

Reply:
Done